### PR TITLE
chore: drop support for PHP 7.3 

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,20 +9,20 @@
     "docs": "https://googleapis.github.io/google-auth-library-php/main/"
   },
   "require": {
-    "php": "^7.1||^8.0",
-    "firebase/php-jwt": "^5.5||^6.0",
+    "php": "^7.4||^8.0",
+    "firebase/php-jwt": "^6.0",
     "guzzlehttp/guzzle": "^6.2.1|^7.0",
-    "guzzlehttp/psr7": "^1.7|^2.0",
+    "guzzlehttp/psr7": "^2.0",
```
Comment (vishwarajanand) [inline composer.json:15]: Seems like older guzzle versions suffer from security vulnerabilities: https://packagist.org/packages/guzzlehttp/psr7/advisories?version=7023692
```suggestion
    "guzzlehttp/psr7": "^2.4.5",
```

File: composer.json [hunk 2]
```diff
@@ -9,20 +9,20 @@
...
     "psr/http-message": "^1.0||^2.0",
```
Comment (vishwarajanand) [inline composer.json:16]: [Optional] Seems [v1.0](https://packagist.org/packages/psr/http-message#1.0) supports php<7.4, so maybe we are better with 1.1
```suggestion
    "psr/http-message": "^1.1||^2.0",
```

File: tests/Credentials/ServiceAccountCredentialsTest.php [hunk 3]
```diff
@@ -26,30 +26,31 @@
 use Google\Auth\CredentialsLoader;
 use Google\Auth\OAuth2;
 use GuzzleHttp\Psr7;
+use GuzzleHttp\Psr7\Response;
```
Comment (vishwarajanand) [inline tests/Credentials/ServiceAccountCredentialsTest.php:29]: all imports here need to be sanitized. Some of them are not used at all hence leading to static check failures.

File: composer.json (hunk 1 above)
Comment (bshaffer) [reply composer.json]: good catch! updated.

File: composer.json (hunk 2 above)
Comment (bshaffer) [reply composer.json]: SGTM

**Review by vishwarajanand — APPROVED** [review]

**Pull request merged**